### ISSUE TITLE
test that info on mouse hover works for keyboard

### ISSUE THREAD

**Issue by jasonkiss** [GOVTNZ/web-standards#5]
While you’re testing interactive element, check that any information presented when you hover your mouse over the element also appears when the element gets keyboard focus.
Record a failure of keyboard functionality if it doesn’t. 

WCAG SC 2.1.1 is about functionality of content, which includes the "outcomes achievable through user action". If hovering over an element reveals meaningful extra info not already in the element, and this functionality is not available via keyboard, I'd call that a 2.1.1 failure.

Many a CMS reproduce link text in the link's title attribute, which is problematic for a few reasons. But the info in these tooltips is repetitive and not meaningful. In these cases where the tooltip's not available via keyboard, I wouldn't call it a failure since there's no supplemental/meaningful info being provided via mouse hover.
